.drone.yml

```diff
@@ -3,21 +3,15 @@ pipeline:
     image: ensime/ensime:v2.x-cache
     pull: true
     environment:
-      - SCALA_VERSION=2.10.6
+      - SCALA_VERSION=2.12.2
```

Comment:
we should just delete this

Reply:
remove env var? even in sbt call?

Reply:
yup, it is decided by sbt for us I should never have done this